sqldelight-compiler/src/main/kotlin/com/squareup/sqldelight/core/compiler/QueryGenerator.kt

```diff
@@ -206,6 +221,15 @@ abstract class QueryGenerator(private val query: BindableQuery) {
     return result.build()
   }
 
+  private val IntermediateType.nameCapitalized: String
+    get() = name.replaceFirstChar {
+      if (it.isLowerCase()) {
+        it.titlecase(Locale.getDefault())
```

Comment:
The above comment should eliminate the need for this function, but in general you want to use a stable locale like `US` rather then the default since that one will vary on each machine and produce different values.